exercises/http_uppercaserer/exercise.js

```diff
@@ -50,14 +50,14 @@ exercise = comparestdout(exercise)
 // delayed for 500ms to wait for servers to start so we can start
 // playing with them
 function query (mode) {
-  var exercise = this
+  const exercise = this
 
   function connect (port, stream) {
-    var input = through2()
-    var count = 0
-    var iv
-    var url = 'http://localhost:' + port
-    var req
+    const input = through2()
+    const count = 0
+    const iv
```

Comment:
should I add a semi-colons or use comma separated variable declaration?

Reply:
This looks good